stdlib/public/TensorFlow/CompilerRuntime.swift

```diff
@@ -1071,7 +1099,12 @@ func _TFCEagerExecute(_ op: CTFEOp,
                       _ status: CTFStatus) {
   if _RuntimeConfig.printsDebugLog {
     debugLog("Calling _TFCEagerExecute() over: ")
-    TFE_OpPrintDebugString(op)
+    if let value = getenv("TF_CPP_MIN_LOG_LEVEL"),
+      String(cString: value) == "0" {
+        TFE_OpPrintDebugString(op)
```

Comment:
Random nit on indentation of the body of multiple-line `if`:
```suggestion
      TFE_OpPrintDebugString(op)
```

Reply:
No need to fix in this PR since you've got CI started :)